package.json

```diff
@@ -34,7 +34,8 @@
     "@types/assert": "^1.5.4",
     "@types/ramda": "^0.28.0",
     "@vue/compat": "^3.2.45",
-    "antlr4": "^4.11.0",
+    "antlr4": "4.11.0",
```

Comment:
antlr4 4.12.0 has error, need to use 4.11.0, see https://github.com/antlr/antlr4/issues/4139